src/formatter-settings/utils.ts

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import * as fse from "fs-extra";
+import * as path from "path";
+import * as vscode from "vscode";
+import { instrumentOperation, sendInfo } from "vscode-extension-telemetry-wrapper";
+
+export async function getProfilePath(formatterUrl: string): Promise<string> {
+    const workspaceFolders = vscode.workspace.workspaceFolders;
+    if (workspaceFolders?.length && !path.isAbsolute(formatterUrl)) {
+        for (const workspaceFolder of workspaceFolders) {
+            const filePath = path.resolve(workspaceFolder.uri.fsPath, formatterUrl);
```

Comment:
what will happen if there are multiple workspace folders and they all have the profiles with the same name?

Reply:
Currently, the java formatter form vscode.java supports only one valid profile, which comes from the first workspace. If there are multiple workspace folders, the settings.json will tell you the setting might not be applied.
![workspace](https://user-images.githubusercontent.com/45906942/118758972-9f320400-b8a2-11eb-80f6-d11d2a2f8232.jpg)